application/pm.py

```diff
@@ -149,8 +149,12 @@ def create_project_dir(self):
             copy(self.videopath, video_dest)
 
             progress_msg.setText("Uploading video file...")
-            identifier = api.uploadVideo(self.videopath)
-            update_config_with_sections(get_config_path(), 'info', 'identifier', identifier)
+            try:
+                identifier = api.uploadVideo(self.videopath)
+                update_config_with_sections(get_config_path(), 'info', 'identifier', identifier)
+            except Exception as e:
+                print("Failed to upload the Video")
+                print(str(e))
```

Comment:
Yeah, this is just a small check to prevent seg faults when we are testing and forget to start up a server. It definitely should be improved.